ConsoleGUI/Controls/VerticalScrollPanel.cs

```diff
@@ -12,6 +12,7 @@ namespace ConsoleGUI.Controls
 	public class VerticalScrollPanel : Control, IDrawingContextListener, IInputListener
 	{
 		private DrawingContext _contentContext = DrawingContext.Dummy;
+
```

Comment:
Please preserve the 0 new line spacing between the private backing field and the front property.
It's used in this code base as indication that the backing field should only be accessed from its front property (being it's integral part) and to make sure that no one squeezes in other declarations between the two lines.